domains/jetus.is.a.dev

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A personal is a dev site.",
+  "repo": "https://github.com/Dev-Jetus/register",
+  "owner": {
+    "username": "jetus",
+    "email": "officialjetus@gmail.com",
+    "twitter": "itzjetus"
+  },
+  "record": {
+    "CNAME": "https://github.com/dev-jetus/dev-jetus.github.io"
```

Comment:
`CNAME` has to be a hostname. In this case, I think you meant `dev-jetus.github.io`

Reply:
Can i just leave this for a bit, and i will do this later on.

Reply:
🤦‍♂️ Just change the CNAME to dev-jetus.github.io then you can get the subdomain...